adapters/kosovo.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * This code is responsible for implementing all methods related to fetching
+ * and returning data for a Kosovo sources (HTML page)
+ */
+'use strict';
+
+const moment = require('moment-timezone');
+const request = require('request-promise-native');
+const cheerio = require('cheerio');
+
+const log = require('../lib/logger');
+
+const openaq_parameters = ['pm25', 'pm10', 'no2', 'so2', 'o3', 'co', 'bc'];
+
+const averagingPeriod = { unit: 'hours', value: 1 };
+const attribution = [{ name: 'Kosovo AQ', url: 'http://kosovo-airquality.com/secure/index2.html' }];
+const staticData = {
```

Comment:
From view-source:http://kosovo-airquality.com/secure/Kosovo.html, I believe you can get the exact locations if you want to update these based on that.

Reply:
Cool, ok to hard-code the locations from that page? I assume they are fairly static...;-)

Reply:
Yep!